### DISCUSSION TITLE
How to create multiple users for schools

### DISCUSSION THREAD

**Discussion by gouravmore** [Sunbird-Saral/Community#76] [General]
Hi,

Is there any way to create the multiple users for schools? 

We can see the users list Json in back-end code-base. After importing the golden data - we can use this users for school login - But not able to find routes/APIs to create the users. 

Can you please help us in creating the multiple users for schools.

**Comment by apurvabisht97** [comment]
hi yes you can create multiple users for school by giving same schoolId .

there is no API for creating users as of now . In users.json you can create multiple users with same schoolId.

**Reply by gouravmore** [reply]
Thanks @apurvabisht97 For the reply.

We have tried to create multiple users by updating the users.json and importing the data into the database.
But when we tried to log in to the app using this users credentials - it's throwing the error 
``` "error": "School Id or Password is not correct."```

@apurvabisht97 Could you please mention the steps here to add multiple users for schools?

**Comment by gouravmore** [comment]
@apurvabisht97 any update on this?  Could you please mention the steps here to add multiple users for schools?

**Reply by apurvabisht97** [reply]
currently we have one users.json under data folder .. so there you can add new users for one school .
